addons/web/static/src/views/fields/file_handler.js

```diff
@@ -88,4 +88,5 @@ FileUploader.props = {
     fileUploadClass: { type: String, optional: true },
     acceptedFileExtensions: { type: String, optional: true },
     slots: { type: Object, optional: true },
+    disableUploadingText: { type: Boolean, optional: true },
```

Comment:
I prefer the opposite: `props.showUploadingText` whose default value is `true`, and we pass `false`.
This prevents double negation, which makes it a bit easier to read and understand.